src/main/java/com/zipcodewilmington/looplabs/IntegerDuplicateDeleter.java

```diff
@@ -5,4 +5,73 @@
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class IntegerDuplicateDeleter extends DuplicateDeleter<Integer> {
+
+    public IntegerDuplicateDeleter(Integer[] intArray) {
+        super(intArray);
+    }
+
+    @Override
+    public Integer[] removeDuplicates(int maxNumberOfDuplications) {
+
+        StringBuilder taco = new StringBuilder();
+
+        for (Integer loopingTaco : array) {
```

Comment:
loopingTaco is a bad name, not only because you're calling it taco, but also because its the values from the array and not the loop itself.